### PR TITLE
chore(webapp): fix all existing Biome errors, add strictTemplate compiler options

### PR DESCRIPTION
**Pull request by irvingoujAtDevolution**
Fixed all biome issues, now we can confidently integrate Biome checking into our CI workflow
Added strictTemplate compiler option, better type safety.

1. Strongly typed WebSession
2. Delete a function (manageScreenSize) that I suppose does nothing (i.e Typescript disagree with it and has no effect on the usage)
3. Removed manual type as much as possible, let TypeScript Infer the types instead.

### CODE REVIEW

File: webapp/biome.json [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+    ]
+  },
   "linter": {
     "rules": {
       "style": {
         "useTemplate": "off",
         "useImportType": "off"
+      },
+      "complexity": {
+        "noUselessConstructor": "off"
```
Comment (irvingoujAtDevolution) [inline webapp/biome.json:18]: Angular sometimes need empty constructor

**Review by kristahouse — APPROVED** [review]
LGTM! Thanks!

**Review by CBenoit — APPROVED** [review]
Nice! By the way, do we enforce biome & ci in the CI?

**Comment by irvingoujAtDevolution** [comment]
> Nice! By the way, do we enforce biome & ci in the CI?

Not yet, will do so in the next PR

**Pull request merged**